bpf/lib/conntrack.h

```diff
@@ -327,10 +327,15 @@ __ct_lookup(const void *map, struct __ctx_buff *ctx, const void *tuple,
 			}
 
 			*monitor = TRACE_PAYLOAD_LEN;
-			if (ct_entry_alive(entry))
-				break;
-			__ct_update_timeout(entry, bpf_sec_to_mono(CT_CLOSE_TIMEOUT),
+
+			if (ct_entry_alive(entry)) {
+				__ct_update_timeout(entry, bpf_sec_to_mono(CT_CLOSING_TIMEOUT),
+					    dir, seen_flags, CT_REPORT_FLAGS);
+			} else {
+				__ct_update_timeout(entry, bpf_sec_to_mono(CT_CLOSE_TIMEOUT),
 					    dir, seen_flags, CT_REPORT_FLAGS);
+			}
+
```

Comment:
Let's condense this down a bit:

```suggestion
			lifetime = ct_entry_alive(entry) ? CT_CLOSING_TIMEOUT : CT_CLOSE_TIMEOUT;
			__ct_update_timeout(entry, bpf_sec_to_mono(lifetime), dir, seen_flags, CT_REPORT_FLAGS);
```